crates/biome_js_analyze/src/lint/nursery/no_static_element_interactions.rs

```diff
@@ -0,0 +1,257 @@
+                            false
+                        }
+                    })
+                } else {
+                    false
+                }
+            });
+
+            if !has_interactive_handler {
+                return None;
+            }
+        } else {
+            return None;
+        }
+
+        if node
+            .find_attribute_by_name("aria-hidden")
+            .map_or(false, |attr| {
+                attr.as_static_value()
+                    .map_or(true, |val| val.text() == "true")
+            })
+        {
+            return None;
+        }
+
+        let is_valid_element = match element_name {
+            "section" => ["aria-label", "aria-labelledby"].iter().any(|&attr_name| {
+                node.find_attribute_by_name(attr_name)
+                    .map_or(false, |attr| {
+                        attr.as_static_value()
+                            .map_or(false, |val| !val.text().is_empty())
+                    })
+            }),
+            "a" => node.find_attribute_by_name("href").map_or(false, |attr| {
+                attr.as_static_value()
+                    .map_or(false, |val| !val.text().is_empty())
+            }),
+            _ => {
+                // In Biome-managed elements, whether an element is interactive is determined by the is_not_interactive_element function.
+                (!aria_roles.is_not_interactive_element(element_name, attributes.clone())
+                    // Except for elements invalid in the eslint-plugin-jsx-a11y test case.
+                    && !is_invalid_element(element_name))
+                    // Also, elements marked as valid in eslint-plugin-jsx-a11y are marked as “valid” even if they are marked as invalid in Biome.
+                    || is_valid_element(element_name)
+            }
+        };
+
+        if is_valid_element {
+            return None;
+        }
+
+        if let Some(attr) = node.find_attribute_by_name("role") {
+            let role_value = attr.as_static_value()?;
+
+            if aria_roles.is_role_interactive(role_value.text()) {
+                return None;
+            }
+        }
+
+        Some(())
+    }
+
+    fn diagnostic(ctx: &RuleContext<Self>, _: &Self::State) -> Option<RuleDiagnostic> {
+        let node = ctx.query();
+        Some(RuleDiagnostic::new(
+            rule_category!(),
+            node.range(),
+            markup! {{"Static Elements should not be interactive."}},
+        ).note(
+            markup! {{"To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value."}}
+        ))
+    }
+}
+
+/// Checks if the given element name is considered invalid, inspired by eslint-plugin-jsx-a11y
+fn is_invalid_element(element_name: &str) -> bool {
+    match element_name {
+        // These cases are interactive with the is_not_interactive_element method,
+        // but is an invalid test case element for eslint-plugin-jsx-a11y.
```

Comment:
@ematipico 
Thank you for your comment.
Your comment is correct, but we will hold it in abeyance because the way we respond will depend on the answers to the following questions.
I will address this when this method continues to be needed.